prosessering-core/src/main/kotlin/no/nav/familie/prosessering/internal/TaskStepExecutorService.kt

```diff
@@ -58,13 +62,20 @@ class TaskStepExecutorService(
     @Scheduled(fixedDelayString = "\${prosessering.fixedDelayString.in.milliseconds:30000}")
     fun pollAndExecute() {
         if (!enabled) return
+        if (!isRunning.compareAndSet(false, true)) {
+            log.debug("Kjører allerede")
+        }
 
-        while (true) {
-            if (isShuttingDown) {
-                log.info("Shutting down, does not start new pollAndExecuteTasks")
-                return
+        try {
+            while (true) {
+                if (isShuttingDown) {
+                    log.info("Shutting down, does not start new pollAndExecuteTasks")
+                    return
+                }
+                if (!pollAndExecuteTasks()) return
```

Comment:
hvorfor trekkes denne inn i while(true)?

Reply:
Den fantes der fra før, det har bare blitt noen flere spaces pga wrap med try catch :) 
Den er der fra før for å kunne håndtere at man skal prosessere forløpende (continuousRunning)